resources/configurationFiles/analysisQC.xml

```diff
@@ -566,6 +565,18 @@
         <filename class='ChromosomeDiffValueFile' derivedFrom='BamFile' pattern='${outputAnalysisBaseDirectory}/${structuralVariationOutputDirectory}/${sourcefileAtomic}_DiffChroms.png_qcValues.txt'/>
         <filename class='CoverageTextFile' onMethod='BamFileGroup.mergeAndRemoveDuplicatesSlim' selectiontag="readBinsCoverage" pattern='${outputAnalysisBaseDirectory}/${coverageOutputDirectory}/${sample}_${pid}_readCoverage_${cvalue,name="WINDOW_SIZE",default="1"}kb_windows.txt'/>
         <filename class='CoverageTextFile' onMethod='BamFileGroup.mergeAndRemoveDuplicatesSlim' selectiontag="genomeCoverage" pattern='${outputAnalysisBaseDirectory}/${coverageOutputDirectory}/${sourcefileAtomicPrefix,delimiter="_"}.DepthOfCoverage_Genome.txt'/>
+
+        <filename class='TextFile' onScriptParameter='accelerated_alignAndPairSlim:FILENAME_GROUPED_GENOME_COVERAGE'
```

Comment:
Why don't you put this to the tool parameter in the tool definition itself?

Reply:
I follow the convention for this workflow.

Reply:
Also, I had cases in the BamToFastqWorkflow, where the association with the tool did not work.